packages/editor/src/components/post-author/check.js

```diff
@@ -16,10 +16,9 @@ import PostTypeSupportCheck from '../post-type-support-check';
 
 export function PostAuthorCheck( {
 	hasAssignAuthorAction,
-	authors,
 	children,
 } ) {
-	if ( ! hasAssignAuthorAction || authors.length < 2 ) {
```

Comment:
What's the rationale for this?

Reply:
hmm, possibly removed by mistake, will check

Reply:
This was related to having the authors reset by search which is now fixed with the query key.  Reverted these changes and verified search with empty search doesn't make the field disappear as before.